packages/retail-ui/components/Popup/Popup.tsx

```diff
@@ -324,7 +330,8 @@ export default class Popup extends React.Component<PopupProps, PopupState> {
     }
   };
 
-  private renderContent(location: PopupLocation) {
+  private renderContent() {
+    const location = this.state.location || this.getDummyLocation();
```

Comment:
Может бы сделать вот так?
```suggestion
    const location = this.state.location || DUMMY_LOCATION;
```

Reply:
точно! done